.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: objective-c
+osx_image: xcode63
+xcode_workspace: fh-ios-sdk.xcworkspace
+xcode_scheme: FH
+xcode_sdk: iphonesimulator
+
+before_install:
+    - gem install cocoapods --no-rdoc --no-ri --no-document --quiet
```

Comment:
let's further add the notification step once we are about to merge [like here](https://github.com/aerogear/aerogear-ios-http/blob/master/.travis.yml#L10-L18)